go.mod

```diff
@@ -10,6 +10,10 @@ require (
 	github.com/line/gorocksdb v0.0.0-20210406043732-d4bea34b6d55
 	github.com/stretchr/testify v1.7.0
 	github.com/syndtr/goleveldb v1.0.1-0.20200815110645-5c35d600f0ca
-	go.etcd.io/bbolt v1.3.5
-	google.golang.org/grpc v1.35.0
+	go.etcd.io/bbolt v1.3.6
+	google.golang.org/grpc v1.42.0
 )
+
+// FIXME: gorocksdb bindings for OptimisticTransactionDB are not merged upstream, so we use a fork
+// See https://github.com/tecbot/gorocksdb/pull/216
+// replace github.com/tecbot/gorocksdb => github.com/roysc/gorocksdb v1.1.1
```

Comment:
Why is this line commented out?

Reply:
Because we use line/gorocksdb, not tecbot/gorocksdb or roysc/gorocksdb.